linux/kernel/building.md

```diff
@@ -103,10 +103,21 @@ You can either do this using VirtualBox (or VMWare) on Windows, or install it di
 
 To build the sources for cross-compilation, make sure you have the dependencies needed on your machine by executing:
 ```bash
-sudo apt install git bc bison flex libssl-dev make libc6-dev libncurses5-dev crossbuild-essential-armhf
+sudo apt install git bc bison flex libssl-dev make libc6-dev libncurses5-dev
 ```
+
 If you find you need other things, please submit a pull request to change the documentation.
 
+#### Install the 32-Bit toolchain for a 32-Bit Kernel
```

Comment:
On a stylistic note, I don't like the random capitalisation of certain words such as "Bit", "Kernel" and "Config".

Reply:
Resolved in b66bbe7